docs/docs/launch-manual/sections/behind-pulsar/sections/maintaining-your-packages.md

```diff
@@ -0,0 +1,132 @@
+## Maintaining Your Packages
+
+<!--This entire file needs to be checked for accuracy and content once the new
+backend supports this fully.
```

Comment:
If everything has gone well, this whole section should work with no issues on the new backend. Great care has been taken to ensure, to the best of my ability, that it's fully compatible with PPM with zero changes

Reply:
If that is the case then and you are sure there isn't anything in this file that is incorrect then I'm happy to remove the comment and just treat it as final.

Reply:
Yeah it looks good to me. What we could do, just in case is to say these features are in beta. Just to set expectations properly, and if over time we see that transitions have gone smoothly we remove it.

But as for the intended behavior this looks perfect.

Reply:
Thats a good point, maybe we could add a new container block (like the danger/warning ones) to point this out when there is still work ongoing in the main repo to support it.

Reply:
Yeah @Daeraxa I'd love this idea. Basically that below is the intended functionality, while many of the features and functionality are still undergoing work to support these features with stability.

Something like that, but would fit perfectly.

Reply:
I've removed the comment and added a new tips container to the top. When this is merged I'll add it to our common containers list.